src/EnergyPlus/EconomicLifeCycleCost.cc

```diff
@@ -460,11 +460,11 @@ void GetInputLifeCycleCostRecurringCosts(EnergyPlusData &state)
     NumArray.allocate(NumNums);
     AlphaArray.allocate(NumAlphas);
     elcc->numRecurringCosts = state.dataInputProcessing->inputProcessor->getNumObjectsFound(state, CurrentModuleObject);
-    elcc->RecurringCosts.allocate(elcc->numRecurringCosts);
-    for (iInObj = 1; iInObj <= elcc->numRecurringCosts; ++iInObj) {
+    elcc->RecurringCosts.resize(elcc->numRecurringCosts);
+    for (iInObj = 0; iInObj < elcc->numRecurringCosts; ++iInObj) {
         state.dataInputProcessing->inputProcessor->getObjectItem(state,
                                                                  CurrentModuleObject,
-                                                                 iInObj,
+                                                                 iInObj+1,
                                                                  AlphaArray,
                                                                  NumAlphas,
                                                                  NumArray,
```

Comment:
I had to use +1 for the third argument in the getObjectItem() function so the vector does not go out of bounds.